### PR TITLE
Feat doc tests

### PR DESCRIPTION
**Pull request by benib**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,97 @@
...
+
+```bash
+$ nvm use
+$ npm install
+$ npm run build
+```
+
+## Configuration
+
+No configuration is needed for this tool.
```
Comment (manuelroth) [inline README.md]: @philipkueng @benib shall we remove this section if no configuration is needed?- Or keep it to document that there are no env vars needed?- What do you think @philipkueng

File: README.md [hunk 2]
```diff
@@ -1,22 +1,97 @@
...
+
+## Development
+
```
Comment (manuelroth) [inline README.md]: @philipkueng did the development section a bit different in q-table -> https://github.com/nzzdev/Q-table/tree/feat-documentation#development

## Development

Install the [Q cli](https://github.com/nzzdev/Q-cli) and start the Q dev server:

```
$ Q server
```

Run the Q tool:
```
$ node index.js
```

File: README.md (hunk 1 above)
Comment (philipkueng) [reply README.md]: I think it should be removed when not used.

File: README.md [hunk 3]
```diff
@@ -1,22 +1,97 @@
...
+[to the top](#table-of-contents)
+
+## Features
+
+There is only one endpoint to get `renderingInfo` in this tool.
+
+### `/rendering-info/html-static`
+
+This endpoint returns a renderingInfo object with static HTML/CSS/SVG that is fully responsive.
+
```
Comment (philipkueng) [inline README.md]: add after features
`[to the top](#table-of-contents)`

File: README.md [hunk 4]
```diff
@@ -1,22 +1,114 @@
...
+[to the top](#table-of-contents)
+
+## Testing
+
+The testing framework used in this repository is [Code](https://github.com/hapijs/code).
+
+Run the tests:
+
+```
+$ npm run test
```
Comment (manuelroth) [inline README.md]:
```suggestion
npm run test
```

File: README.md [hunk 5]
```diff
@@ -1,22 +1,114 @@
...
+This endpoint returns a renderingInfo object with static HTML/CSS/SVG that is fully responsive.
+
+### Options
+
+#### hideUpdatedDate
+
+This option is only available for users with the `expert-election-votes` roles. It hides the updated date and is mainly used when you enter historic data where the last updated date makes no sense.
+
+[to the top](#table-of-contents)
+
```
Comment (philipkueng) [inline README.md]: Add `Display options`

File: README.md (hunk 1 above)
Comment (benib) [reply README.md]: we decided to keep it

**Pull request merged**